VerbalExpressions.js

```diff
@@ -76,66 +51,30 @@ class VerbalExpression extends RegExp {
 
     // Rules //
 
-    /**
-     * Control start-of-line matching
-     * @param {boolean} [enable=true] whether to enable this behaviour
-     * @returns {VerbalExpression} recompiled instance of VerbalExpression
-     * @memberof VerbalExpression
-     */
     startOfLine(enable = true) {
         this._prefixes = enable ? '^' : '';
         return this.add();
     }
 
-    /**
-     * Control end-of-line matching
-     * @param {boolean} [enable=true] whether to enable this behaviour
-     * @returns {VerbalExpression} recompiled instance of VerbalExpression
-     * @memberof VerbalExpression
-     */
     endOfLine(enable = true) {
         this._suffixes = enable ? '$' : '';
         return this.add();
     }
 
-    /**
-     * Look for the value passed
-     * @param {(string|RegExp|number)} value value to find
-     * @returns {VerbalExpression} recompiled instance of VerbalExpression
-     * @memberof VerbalExpression
-     */
     then(value) {
         value = VerbalExpression.sanitize(value);
         return this.add(`(?:${value})`);
     }
 
-    /**
-     * Alias for then() to allow for readable syntax when then() is the first method in the chain.
-     * @param {(string|RegExp|numer)} value value to find
-     * @returns {VerbalExpression} recompiled instance of VerbalExpression
-     * @memberof VerbalExpression
-     */
```

Comment:
can we still keep the jsdoc comments here even if they are not used for generating the /docs ?

Reply:
If you guys insist, sure.

Reply:
@ionutvmi done.